packages/client/src/components/reactions-viewer.reaction.vue

```diff
@@ -144,6 +144,10 @@ export default defineComponent({
 		> span {
 			color: var(--fgOnAccent);
 		}
+
+		> .mk-emoji {
```

Comment:
It is better for parents not to know what classes their children have.

Reply:
Without that, the only possible solution would be to add shadow to all emojis. I'm not sure if that is a good idea.